### PR TITLE
CMET feature details

### PR DESCRIPTION
**Pull request by Aboli-msft**

### CODE REVIEW

File: 06-Customizing AzTS for your org/Extending AzTS/UpdateControlMetadata.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+**Step 5:** Click on 'Close' button.
+
+**Step 6:** Click on **Action** and select **Update** option.
+![Update](../../Images/06_ExtendingAzTS_UpdateControl_2.png)
+
+**Step 7:** Add appropriate comments and click on 'Confirm'.
+![Update](../../Images/06_ExtendingAzTS_UpdateControlMetadata.png)
+
+Post this, you will get success message. Now you are good to **close** 'Update Control Metadata' window.
```
Comment (TarunKrShukla) [inline 06-Customizing AzTS for your org/Extending AzTS/UpdateControlMetadata.md:29]: NIT: In this image comment is 'test', please add some more appropriate text as comment to educate users.

File: 06-Customizing AzTS for your org/Extending AzTS/Prerequisites.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
-## Prerequisite AzTS configurations to add new control
-Below mentioned appsettings are required before adding new controls:
+## Prerequisite AzTS configurations to enable Control Medatadata Editor tool(CMET)
+Below mentioned appsettings are required before enabling CMET:
```
Comment (asimks) [inline 06-Customizing AzTS for your org/Extending AzTS/Prerequisites.md:2]: before enabling CMET -> for enabling CMET

File: 06-Customizing AzTS for your org/Extending AzTS/Readme.md [hunk 3]
```diff
@@ -1,16 +1,21 @@
 > The Azure Tenant Security Solution (AzTS) was created by the Core Services Engineering & Operations (CSEO) division at Microsoft, to help accelerate Microsoft IT's adoption of Azure. We have shared AzTS and its documentation with the community to provide guidance for rapidly scanning, deploying and operationalizing cloud resources, across the different stages of DevOps, while maintaining controls on security and governance.
 <br>AzTS is not an official Microsoft product – rather an attempt to share Microsoft CSEO's best practices with the community.
 
-# Extending AzTS
+# Extending AzT
```
Comment (asimks) [inline 06-Customizing AzTS for your org/Extending AzTS/Readme.md:4]: Extending AzT -> Extending AzTS

File: 06-Customizing AzTS for your org/Extending AzTS/Readme.md [hunk 4]
```diff
@@ -1,16 +1,21 @@
...
+Azure Tenant Security Solution (AzTS) provides capability to add new controls(for existing services supported by AzTS) to customize the AzTS for your organization as per your need.
+Currently you can extend AzTS controls set by either adding a new control based on custom Azure Policy or based on Microsoft Defender for Cloud assessment using Control Metadata Editor Tool. 
 
-## Add new control for existing SVT
+## Enabling Control Medatadata Editor tool
+This feature is not enabled by default. To enable this feature for your AzTS setup, please follow steps mentioned [here](Prerequisites.md#prerequisite-azts-configurations-to-enable-control-medatadata-editor-toolcmet).
 
-Azure Tenant Security Solution (AzTS) provides capability to add new controls(for existing services supported by AzTS) to customize the AzTS for your organization as per your need.
-Currently you can extend AzTS controls set by either adding a new control based on custom Azure Policy or based on Microsoft Defender for Cloud assessment.
+## Add new control for existing SVT
```
Comment (asimks) [inline 06-Customizing AzTS for your org/Extending AzTS/Readme.md:11]: existing SVT -> existing Services

**Review by TarunKrShukla — APPROVED** [review]
looks good

**Review by asimks — APPROVED** [review]

**Pull request merged**